sdk/monitor/azure-monitor-opentelemetry/azure/monitor/opentelemetry/_configure.py

```diff
@@ -124,7 +125,7 @@ def _setup_logging(configurations: Dict[str, ConfigurationValue]):
     logger_provider = LoggerProvider(resource=resource)
     set_logger_provider(logger_provider)
     log_exporter = AzureMonitorLogExporter(**configurations)
-    log_record_processor = BatchLogRecordProcessor(
+    log_record_processor = _AzureMonitorLogRecordProcessor(
```

Comment:
Need DisableTraceBasedSamplingForLogs option.

Reply:
Vishwesh says no env var is planned. Only a param. How about "disable_trace_based_sampling_for_logs"?